lib/tasks/build.js

```diff
@@ -61,5 +62,9 @@ module.exports = Task.extend({
 
         throw err;
       });
+  },
+
+  onInterrupt: function() {
+    return RSVP.Promise.reject(1);
```

Comment:
Actually the more idiomatic way of doing this is probably just to call `process.exit(code)` and it saves us from having to figure out the various different cases in which we exit, possibly with a `process.exit`-provided code, and also with a handler-provided code.